tests/integration/init/schemas/schemas_test_data_setup.py

```diff
@@ -128,17 +115,28 @@ def _create_config_file(self, profile, region):
             file.write(config_file_content)
         return custom_config
 
-    def _create_cred_file(self, profile, access_key, secret_key):
-        cred_file_content = "[default]\naws_access_key_id = {1}\naws_secret_access_key = {2}\n"
+    def _create_cred_file(self, profile, access_key, secret_key, session_token=None):
+        cred_file_content = self._create_cred_profile("default", access_key, secret_key, session_token)
         if profile != DEFAULT:
-            cred_file_content += "\n[{0}]\naws_access_key_id = {1}\naws_secret_access_key = {2}"
-        cred_file_content = cred_file_content.format(profile, access_key, secret_key)
+            cred_file_content += f"\n{self._create_cred_profile(profile, access_key, secret_key, session_token)}"
         custom_cred = os.path.join(self.config_dir, "customcred")
         print("Writing custom creds to {}".format(custom_cred))
         with open(custom_cred, "w") as file:
             file.write(cred_file_content)
         return custom_cred
 
+    def _create_cred_profile(self, profile_name, access_key, secret_key, session_token=None):
+        """
+        Method to create aws credentials entry similar to ~/.aws/credentials file format.
+        """
+        cred_profile_content = f"""
+[{profile_name}]
+aws_access_key_id = {access_key}
+aws_secret_access_key = {secret_key}
+"""
+        cred_profile_content += f"aws_session_token={session_token}\n"
```

Comment:
Why not have this in the `cred_profile_content` above?

Reply:
Ohh that is a miss, there is supposed to be an if in there to add this only if session_token is passed. Lost somewhere in merging. I'll fix it.